shoplist/src/main/java/com/netguru/codereview/network/ShopListApiMock.kt

```diff
@@ -0,0 +1,42 @@
+package com.netguru.codereview.network
+
+import com.netguru.codereview.network.model.ShopListItemResponse
+import com.netguru.codereview.network.model.ShopListResponse
+import kotlinx.coroutines.coroutineScope
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.flow
+
+class ShopListApiMock : ShopListApi {
+    override suspend fun getShopLists(): List<ShopListResponse> =
+        coroutineScope {
```

Comment:
there is no need for using  "coroutineScope"  
or we can use "withContext()"  if we need to switch thread